files/initramfs-tools/arista-convertfs.j2

```diff
@@ -105,8 +126,8 @@ if ! wait_for_root_dev; then
     echo "Error: timeout in waiting for $root_dev"
     exit 1
 fi
-blkid | grep "$root_dev.*vfat" -q || exit 0
 
+blkid | grep "$root_dev.*vfat" -q || create_varlog_file
```

Comment:
I know the code is working. We can still make the code more readable and less error prone.